ci/builders/README.md

```diff
@@ -1,69 +1,55 @@
-* **Builder** - a combination of configuration and recipes used with a given commit to build artifacts and test them.
-* **Build** - a builder running with specific properties, repository and commit.
-* **[Gn](https://gn.googlesource.com/gn/)**, a meta-build system that generates build files for [Ninja](https://ninja-build.org/).
-* **[Ninja](https://ninja-build.org)**, Ninja is a small build system with a focus on speed.
-* **CAS**, a service that stores arbitrary binary blobs addressed by (hash of) their content. It is specialized for low latency, high volume query/read/write operations.
-
-## OVERVIEW
-
-To describe engine builds in a generic and scalable way we are creating a build definition language that supports all the current build use cases. The definition language is using json to describe builds, tests, archives, and dependency relations.
-
-This definition language will be used by generic recipes to trigger independent build units in different machines using cas for intermediate storage and an orchestrator recipe that collects and integrates the multiple pieces.
-
-On top of the current use cases the definition language also supports configurations for code signing and describing the artifacts explicitly. These two new use cases are required for the implementation of SLSA requirements and the implementation of a single command Flutter SDK release.
-
-The definition language makes heavy use of GN and Ninja to move most of the build/test/archive logic to the flutter/engine repository. This will help us remove most of the logic from recipes and make the recipes implementation generic and reusable by Dart & Flutter teams.
-
-#### Non-goals
-
-Using a format different from json to describe the engine builds.
+# Flutter Engine Build Definition Language
+
+The ***Flutter Engine Build Definition Language*** describes a build on CI
+by defining a combination of *sub-builds*, *archives*, *generators* and *dependencies*. It
+makes it simple to shard sub-builds by mapping build inputs to workflows, and listing
+the sub-build-generated artifacts explicitly. The Build Definition Language, Engine
+Recipes V2 and the generation of artifacts using GN+Ninja set the groundwork
+for efficient builds with dependency reusability.  
+
+**Author: Godofredo Contreras (godofredoc)**\
+**Go Link: flutter.dev/go/engine-build-definition-language**\
+**Created:** 01/2023   /  **Last updated:** 04/2023
+
+## Glossary
+
+* **[recipes](https://github.com/luci/recipes-py)** - domain specific
+language for specifying sequences of subprocess calls in a cross-platform and
+testable way.
+* **Generator** - scripts in Dart, python or bash that combine the output of
+sub-builds to generate artifacts.
+* **Builder** - a combination of configuration, recipes and a given commit to
+build and test artifacts.
+* **Build** - a builder running with specific properties, repository and
```

Comment:
So these two (builder and build) seem to be heavily overloaded. Build is specified a single top level build that displays in the flutter dashboard and is expressed as a single file containing multiple sub builds.

A builder is also referred to as the runner of this top level build and also a runner of a sub build, which we may want to specify somewhere in here. So builder refers to two different parts of an engine_v2 build.